packages/components/page-header/src/page-header.vue

```diff
@@ -5,22 +5,23 @@
     </div>
     <div :class="ns.e('header')">
       <div :class="ns.e('left')">
-        <div
-          v-if="icon || $slots.icon"
-          :aria-label="title || t('el.pageHeader.title')"
-          :class="ns.e('icon')"
-          role="button"
-          tabindex="0"
-          @click="handleClick"
-        >
-          <slot name="icon">
-            <el-icon v-if="icon">
-              <component :is="icon" />
-            </el-icon>
-          </slot>
-        </div>
-        <div :class="ns.e('title')">
-          <slot name="title">{{ title || t('el.pageHeader.title') }}</slot>
+        <div :class="ns.e('back')" @click="handleClick">
+          <div
+            v-if="icon || $slots.icon"
+            :aria-label="title || t('el.pageHeader.title')"
+            :class="ns.e('icon')"
+            role="button"
+            tabindex="0"
```

Comment:
The `role` and `tabindex` attribute should probably be moved with the click event

Reply:
This is indeed a Bug. Now, it will not be possible to trigger the back event by pressing Tab and Enter keys.

Reply:
I moved this to the parent element so this will be triggered correctly now.